static/src/javascripts/bootstraps/ophan.standalone.ts

```diff
@@ -0,0 +1,8 @@
+// Monkey patch to facilitate the removal of ophan tracking from the commercial bundle sent to DCR.
+
+const ophan: Ophan = window.guardian.ophan;
```

Comment:
i'm a bit unclear why you have this? what happens if this does not exist?

Reply:
This file isn’t currently used, so I’ll remove it for now.